src/richie/apps/courses/templates/courses/cms/category_detail.html

```diff
@@ -5,12 +5,16 @@
 {% with category=current_page.category header_level=2 %}
 <div class="category-detail">
   <div class="category-detail__banner">
-    {% placeholder "banner" %}
+    {% placeholder "banner" or %}
+      <div class="category-detail__banner__empty">{% trans "Banner" %}</div>
+    {% endplaceholder %}
   </div>
 
   <div class="category-detail__logo">
     {% with width=216 height=120 %}
-      {% placeholder "logo" %}
+      {% placeholder "logo" or %}
+        <div class="category-detail__logo__empty">{% trans "Logo" %}</div>
+      {% endplaceholder %}
     {% endwith %}
```

Comment:
On a Category page:
- hide "related news" section if no related news
- hide "related courses" section if no related courses

Reply:
Did the same in Org detail when there is no course it don't show the "Related courses" block.